importer/views.py

```
@@ -207,41 +211,43 @@ class BookListView(TemplateView):
 
-        redirect_url = self.request.META.get('HTTP_REFERER', '')
-        if 'match' in redirect_url:
+        redirect_url = self.request.META.get("HTTP_REFERER", "")
+        if "match" in redirect_url:
             context.update({"default_view": "processing"})
 
-        for key, books in filter(lambda item: 'books' in item[0], kwargs.items()):
-            context.update(
-                {key: list(zip(books, self.calcBookLength(list(books))))})
+        for key, books in filter(lambda item: "books" in item[0], kwargs.items()):
+            context.update({key: list(zip(books, self.calcBookLength(list(books))))})
```

Ruff (0.15.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)